modules/triggers-supported-filter-dialects.adoc

```diff
@@ -0,0 +1,150 @@
+
+.Example of `prefix` filter dialect
+[source,terminal]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  ...
+spec:
+  ...
+  filters:
+    - prefix:
+        type: com.github.
+----
+
+[id="triggers-filter-suffix-dialect_{context}"]
+== `suffix` filter dialect
+
+The `suffix` dialect filters events where a CloudEvent attribute's string value ends with the specified string. This comparison is case-sensitive. If the attribute is not a string, the filter converts the attribute to its string representation before matching it to the specified value.
+
+.Example of `suffix` filter dialect
+[source,terminal]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  ...
+spec:
+  ...
+  filters:
+    - suffix:
+        type: .created
+----
+
+[id="triggers-filter-all-dialect_{context}"]
+== `all` filter dialect
+
+The `all` filter dialect requires that all nested filter expressions evaluate to `true` for the event to be processed. If any of the nested expressions return `false`, the event will not be sent to the subscriber.
+
+.Example of `all` filter dialect
+[source,terminal]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  ...
+spec:
+  ...
+  filters:
+    - all:
+        - exact:
+            type: com.github.push
+        - exact:
+            subject: https://github.com/cloudevents/spec
+----
+
+[id="triggers-filter-any-dialect_{context}"]
+== `any` filter dialect
+
+The `any` filter dialect requires at least one of the nested filter expressions to evaluate to `true`. If none of the nested expressions return `true`, the event will not be sent to the subscriber.
+
+.Example of `any` filter dialect
+[source,terminal]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  ...
+spec:
+  ...
+  filters:
+    - any:
+        - exact:
+            type: com.github.push
+        - exact:
+            subject: https://github.com/cloudevents/spec
+----
+
+[id="triggers-filter-not-dialect_{context}"]
+== `not` filter dialect
+
+The `not` filter dialect requires that the nested filter expression evaluates to `false` for the event to be processed. If the nested expression evaluates to `true`, the event will not be sent to the subscriber.
+
+.Example of `not` filter dialect
+[source,terminal]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  ...
+spec:
+  ...
+  filters:
+    - not:
+        exact:
+          type: com.github.push
+----
+
+[id="triggers-filter-cesql-dialect_{context}"]
+== `cesql` filter dialect 
+
+The `cesql` filter dialect uses CloudEvents SQL expressions to filter events. The provided CESQL expression must evaluate to `true` for the event to be processed.
+
+.Example of `cesql` filter dialect
+[source,terminal]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  ...
+spec:
+  ...
+  filters:
+    - cesql: "source LIKE '%commerce%' AND type IN ('order.created', 'order.updated', 'order.canceled')"
+----
```

Comment:
I miss a section like this: 
https://knative.dev/v1.14-docs/eventing/experimental-features/new-trigger-filters/#conflict-with-the-current-filter-field

Not the FAQ, but the `Conflict with the current filter field` header/section

Reply:
Yes, it is a WIP PR and I have added the section now. It will reflect in the PR :)

Reply:
The section is alredy present here: https://82063--ocpdocs-pr.netlify.app/openshift-serverless/latest/eventing/triggers/new-trigger-filters#triggers-conflict-current-filter-field_new-trigger-filters